google-account-plugin/src/com/google/cloud/tools/intellij/stats/GoogleUsageTracker.java

```diff
@@ -96,7 +98,8 @@
           new BasicNameValuePair(IS_NON_INTERACTIVE_KEY, STRING_FALSE_VALUE),
           new BasicNameValuePair(
               UNIQUE_CLIENT_ID_KEY,
-              UpdateChecker.getInstallationUID(PropertiesComponent.getInstance())));
+              UpdateChecker.getInstallationUID(PropertiesComponent.getInstance())),
```

Comment:
@etanshaul 

Do you mean the 'dt' value?

DH never changes so it can be intialized into the base data. DT has data about the current ping which is only known at the time the event is called.